commit 5862b9db06821a6a55570196c054fa4a718b9045
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Feb 17 16:36:12 2022 -0300

    Revisão geral de nmod:appos e do lema “como” (adp, adv...)

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 0729b7d..e82c5b1 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -752,7 +752,7 @@
 11	geomorfológicas	geomorfológico	ADJ	_	Gender=Fem|Number=Plur	9	conj	O	_
 12	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	14	case	O	_
 13	como	como	ADP	_	_	12	fixed	O	_
-14	vales	val	NOUN	_	Gender=Masc|Number=Plur	8	nmod:appos	O	_
+14	vales	vale	NOUN	_	Gender=Masc|Number=Plur	8	nmod:appos	O	_
 15	estruturais	estrutural	ADJ	_	Gender=Masc|Number=Plur	14	amod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	14	conj	O	_
@@ -787,9 +787,9 @@
 42	de	de	ADP	_	_	41	flat:name	O	_
 43	Tibagi	Tibagi	PROPN	_	Number=Sing	41	flat:name	O	_
 44	e	e	CCONJ	_	_	46	cc	O	_
-45	“	"	PUNCT	_	Gender=Masc|Number=Sing	46	punct	O	_
+45	“	"	PUNCT	_	_	46	punct	O	_
 46	Sinclinal	Sinclinal	PROPN	_	Number=Sing	41	conj	O	_
-47	”	"	PUNCT	_	Number=Sing	46	punct	O	_
+47	”	"	PUNCT	_	_	46	punct	O	_
 48	de	de	ADP	_	_	49	case	O	_
 49	Torres	Torres	PROPN	_	Gender=Masc|Number=Sing	46	nmod	O	_
 50	(	(	PUNCT	_	_	51	punct	O	_
@@ -2626,7 +2626,7 @@
 13	continental	continental	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
 14	oblíqua	oblíquo	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
 15	como	como	ADP	_	_	16	case	O	_
-16	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	12	nmod:appos	O	_
+16	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 17-18	da	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -5468,7 +5468,7 @@
 18	expressão	expressão	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
 19	sísmica	sísmico	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
 20	como	como	ADP	_	_	21	case	O	_
-21	refletores	refletor	NOUN	_	Gender=Masc|Number=Plur	18	nmod:appos	O	_
+21	refletores	refletor	NOUN	_	Gender=Masc|Number=Plur	17	xcomp	O	_
 22	superpostos	superpor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	21	acl	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
 24	convexos	convexo	NOUN	_	Gender=Masc|Number=Plur	21	conj	O	_
@@ -7360,7 +7360,7 @@
 27	importante	importante	ADJ	_	Gender=Fem|Number=Sing	28	amod	O	_
 28	atividade	atividade	NOUN	_	Gender=Fem|Number=Sing	26	obj	O	_
 29	como	como	ADP	_	_	30	case	O	_
-30	zona	zona	NOUN	_	Gender=Fem|Number=Sing	28	nmod:appos	O	_
+30	zona	zona	NOUN	_	Gender=Fem|Number=Sing	26	obl	O	_
 31	de	de	ADP	_	_	32	case	O	_
 32	falha	falha	NOUN	_	Gender=Fem|Number=Sing	30	nmod	B=ESTRUTURA_DEPOSICIONAL	_
 33	normal	normal	ADJ	_	Gender=Fem|Number=Sing	32	amod	O	_
@@ -9622,12 +9622,12 @@
 4	offshore	offshore	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
 5	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	6	aux:pass	O	_
 6	identificado	identificar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-7	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
+7	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
 8	alto	alto	NOUN	_	Gender=Masc|Number=Sing	6	nsubj:pass	O	_
 9	estrutural	estrutural	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 10	conhecido	conhecer	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	9	acl	O	_
 11	como	como	ADP	_	_	12	case	O	_
-12	Arco	Arco	PROPN	_	Gender=Masc|Number=Sing	10	nmod:appos	O	_
+12	Arco	Arco	PROPN	_	Gender=Masc|Number=Sing	10	xcomp	O	_
 13	de	de	ADP	_	_	12	flat:name	O	_
 14	Torres	Torres	PROPN	_	Number=Sing	12	flat:name	O	_
 15	,	,	PUNCT	_	_	16	punct	O	_
@@ -10663,7 +10663,7 @@
 9	,	,	PUNCT	_	_	12	punct	O	_
 10	como	como	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	8	nmod:appos	O	_
+12	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	2	nmod:appos	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	amolecimento	amolecimento	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 15	,	,	PUNCT	_	_	17	punct	O	_
@@ -11139,16 +11139,16 @@
 21	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	20	fixed	O	_
 22	,	,	PUNCT	_	_	20	punct	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
-24	leite	leite	NOUN	_	Gender=Masc|Number=Sing	14	conj	O	_
+24	leite	leite	NOUN	_	Gender=Masc|Number=Sing	14	nmod:appos	O	_
 25	,	,	PUNCT	_	_	28	punct	O	_
 26	até	até	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	consistência	consistência	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
+28	consistência	consistência	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
 29	cremosa	cremoso	ADJ	_	Gender=Fem|Number=Sing	28	amod	O	_
 30	,	,	PUNCT	_	_	33	punct	O	_
 31	como	como	ADP	_	_	33	case	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
-33	mel	mel	NOUN	_	Gender=Masc|Number=Sing	24	nmod:appos	O	_
+33	mel	mel	NOUN	_	Gender=Masc|Number=Sing	28	nmod:appos	O	_
 34	(	(	PUNCT	_	_	35	punct	O	_
 35	ABEDA	ABEDA	PROPN	_	Gender=Masc|Number=Sing	4	parataxis	O	_
 36	,	,	PUNCT	_	_	35	flat:name	O	_
@@ -11190,7 +11190,7 @@
 29	tais	tal	DET	_	Gender=Masc|Number=Plur|PronType=Dem	32	case	O	_
 30	como	como	ADV	_	_	29	fixed	O	_
 31	:	:	PUNCT	_	_	32	punct	O	_
-32	estabilizantes	estabilizante	NOUN	_	Gender=Masc|Number=Plur	27	appos	O	_
+32	estabilizantes	estabilizante	NOUN	_	Gender=Masc|Number=Plur	27	nmod:appos	O	_
 33	,	,	PUNCT	_	_	34	punct	O	_
 34	agentes	agente	NOUN	_	Gender=Masc|Number=Plur	32	conj	O	_
 35	melhoradores	melhorador	ADJ	_	Gender=Masc|Number=Plur	34	amod	O	_
@@ -12752,7 +12752,7 @@
 24	,	,	PUNCT	_	_	16	punct	O	_
 25	como	como	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
-27	formação	formação	NOUN	_	Gender=Fem|Number=Sing	18	nmod:appos	O	_
+27	formação	formação	NOUN	_	Gender=Fem|Number=Sing	20	nmod:appos	O	_
 28	de	de	ADP	_	_	29	case	O	_
 29	fissuras	fissura	NOUN	_	Gender=Fem|Number=Plur	27	nmod	O	_
 30	,	,	PUNCT	_	_	31	punct	O	_
@@ -19995,7 +19995,7 @@
 20	transgressivos	transgressivo	ADJ	_	Gender=Masc|Number=Plur	18	amod	O	_
 21	(	(	PUNCT	_	_	23	punct	O	_
 22	transgressive	transgressive	ADJ	_	Gender=Masc|Number=Plur	23	amod	O	_
-23	lags	lag	NOUN	_	Gender=Masc|Number=Plur	18	nmod:appos	O	_
+23	lags	lag	NOUN	_	Gender=Masc|Number=Plur	18	appos	O	_
 24	)	)	PUNCT	_	_	23	punct	O	_
 25	.	.	PUNCT	_	_	9	punct	O	_
 
@@ -26589,7 +26589,7 @@
 32	transgressivo	transgressivo	ADJ	_	Gender=Masc|Number=Sing	30	amod	O	_
 33	(	(	PUNCT	_	_	35	punct	O	_
 34	transgressive	transgressive	ADJ	_	Gender=Masc|Number=Plur	35	amod	O	_
-35	lags	lag	NOUN	_	Gender=Masc|Number=Plur	30	nmod:appos	O	_
+35	lags	lag	NOUN	_	Gender=Masc|Number=Plur	30	appos	O	_
 36	)	)	PUNCT	_	_	35	punct	O	_
 37	(	(	PUNCT	_	_	38	punct	O	_
 38	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	5	parataxis	O	_
@@ -28058,7 +28058,7 @@
 41	Araripe	Araripe	PROPN	_	Number=Sing	38	flat:name	I=BACIA	_
 42	,	,	PUNCT	_	_	44	punct	O	_
 43	como	como	ADP	_	_	44	case	O	_
-44	dinoflagelados	dinoflagelado	NOUN	_	Gender=Masc|Number=Plur	34	nmod:appos	O	_
+44	dinoflagelados	dinoflagelado	NOUN	_	Gender=Masc|Number=Plur	31	nmod:appos	O	_
 45	.	.	PUNCT	_	_	4	punct	O	_
 
 # text = Os níveis das coquinasm que correspondem a SIM, sucedem o intervalo dos folhelhos ricos em concreções calcárias fossilíferas e dinoflagelados.
@@ -29809,7 +29809,7 @@
 26	calcárias	calcário	ADJ	_	Gender=Fem|Number=Plur	25	amod	B=LITOLOGIA	_
 27	,	,	PUNCT	_	_	20	punct	O	_
 28	como	como	ADP	_	_	29	case	O	_
-29	Rhacolepis	Rhacolepis	PROPN	_	Gender=Masc|Number=Sing	25	nmod:appos	O	_
+29	Rhacolepis	Rhacolepis	PROPN	_	Gender=Masc|Number=Sing	14	nmod:appos	O	_
 30	,	,	PUNCT	_	_	31	punct	O	_
 31	Leptolepis	Leptolepis	PROPN	_	Gender=Masc|Number=Sing	29	conj	O	_
 32	,	,	PUNCT	_	_	33	punct	O	_
@@ -31087,7 +31087,7 @@
 13	,	,	PUNCT	_	_	16	punct	O	_
 14	como	como	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
-16	Pré-Sal	Pré-Sal	PROPN	_	Gender=Masc|Number=Sing	9	nmod:appos	O	_
+16	Pré-Sal	Pré-Sal	PROPN	_	Gender=Masc|Number=Sing	12	nmod:appos	O	_
 17-18	no	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
@@ -31309,7 +31309,7 @@
 15	mecânicas	mecânico	ADJ	_	Gender=Fem|Number=Plur	11	conj	O	_
 16	,	,	PUNCT	_	_	18	punct	O	_
 17	como	como	ADP	_	_	18	case	O	_
-18	meios	meio	NOUN	_	Gender=Masc|Number=Plur	10	nmod:appos	O	_
+18	meios	meio	NOUN	_	Gender=Masc|Number=Plur	8	xcomp	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
 21	de	de	ADP	_	_	22	case	O	_
@@ -32267,7 +32267,7 @@
 10	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	8	nmod	B=TIPO_FLUIDO	_
 11	como	como	ADP	_	_	13	case	O	_
 12	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	8	nmod:appos	O	_
+13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	6	xcomp	O	_
 14	maduro	maduro	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
 15	,	,	PUNCT	_	_	6	punct	O	_
 16	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	32	nsubj	O	_
@@ -34194,7 +34194,7 @@
 25	,	,	PUNCT	_	_	28	punct	O	_
 26	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	28	case	O	_
 27	como	como	ADP	_	_	26	fixed	O	_
-28	espaços	espaço	NOUN	_	Gender=Masc|Number=Plur	20	nmod:appos	O	_
+28	espaços	espaço	NOUN	_	Gender=Masc|Number=Plur	22	nmod:appos	O	_
 29	bloqueados	bloquear	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	28	acl	O	_
 30	,	,	PUNCT	_	_	31	punct	O	_
 31	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	28	conj	B=ESTRUTURA_DEPOSICIONAL	_
@@ -42227,7 +42227,7 @@
 19-20	do	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	16	nmod	B=TIPO_FLUIDO	_
+21	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	14	obl:arg	B=TIPO_FLUIDO	_
 22	de	de	ADP	_	_	23	case	I=TIPO_FLUIDO	_
 23	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	21	nmod	I=TIPO_FLUIDO	_
 24	,	,	PUNCT	_	_	27	punct	O	_
@@ -42768,9 +42768,9 @@
 17	ser	ser	AUX	_	VerbForm=Inf	18	cop	O	_
 18	natural	natural	ADJ	_	Gender=Fem|Number=Sing	16	xcomp	O	_
 19	,	,	PUNCT	_	_	22	punct	O	_
-20	como	como	ADP	_	_	22	case	O	_
+20	como	como	CCONJ	_	_	22	cc	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
-22	água	água	NOUN	_	Gender=Fem|Number=Sing	5	nmod:appos	B=TIPO_FLUIDO	_
+22	água	água	NOUN	_	Gender=Fem|Number=Sing	2	conj	B=TIPO_FLUIDO	_
 23-24	do	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
@@ -42786,7 +42786,7 @@
 34	de	de	ADP	_	_	35	case	O	_
 35	sais	sal	NOUN	_	Gender=Masc|Number=Plur	33	nmod	B=LITOLOGIA	_
 36	como	como	ADP	_	_	37	case	O	_
-37	NaCl	NaCl	PROPN	_	Gender=Masc|Number=Sing	33	nmod:appos	O	_
+37	NaCl	NaCl	PROPN	_	Gender=Masc|Number=Sing	35	nmod:appos	O	_
 38	,	,	PUNCT	_	_	39	punct	O	_
 39	KCl	KCl	PROPN	_	Gender=Masc|Number=Sing	37	conj	O	_
 40	ou	ou	CCONJ	_	_	41	cc	O	_
@@ -42879,12 +42879,12 @@
 4	específicos	específico	ADJ	_	Gender=Masc|Number=Plur	1	amod	O	_
 5	,	,	PUNCT	_	_	7	punct	O	_
 6	como	como	ADP	_	_	7	case	O	_
-7	anticorrosivos	anticorrosivo	ADJ	_	Gender=Masc|Number=Plur	1	nmod:appos	O	_
+7	anticorrosivos	anticorrosivo	NOUN	_	Gender=Masc|Number=Plur	1	nmod:appos	O	_
 8	,	,	PUNCT	_	_	9	punct	O	_
 9	traçadores	traçador	NOUN	_	Gender=Masc|Number=Plur	7	conj	O	_
 10	químicos	químico	ADJ	_	Gender=Masc|Number=Plur	9	amod	O	_
 11	,	,	PUNCT	_	_	12	punct	O	_
-12	antiespumantes	antiespumante	ADJ	_	Gender=Masc|Number=Plur	7	conj	O	_
+12	antiespumantes	antiespumante	NOUN	_	Gender=Masc|Number=Plur	7	conj	O	_
 13	,	,	PUNCT	_	_	15	punct	O	_
 14	entre	entre	ADP	_	_	15	case	O	_
 15	outros	outro	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	18	obl	O	_
@@ -42945,7 +42945,7 @@
 
 # text = Como essas rochas superficiais são praticamente inertes ao contato com a água doce, o tratamento químico dispensado ao fluido durante esta fase, não é muito intenso.
 # sent_id = 24-20150122-MONOGRAFIA_0-74
-1	Como	como	ADV	_	_	7	advmod	O	_
+1	Como	como	SCONJ	_	_	7	mark	O	_
 2	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	3	det	O	_
 3	rochas	rocha	NOUN	_	Gender=Fem|Number=Plur	7	nsubj	O	_
 4	superficiais	superficial	ADJ	_	Gender=Fem|Number=Plur	3	amod	O	_
@@ -43437,7 +43437,7 @@
 13	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	16	case	O	_
 14	como	como	ADP	_	_	13	fixed	O	_
 15	em	em	ADP	_	_	16	case	O	_
-16	zonas	zona	NOUN	_	Gender=Fem|Number=Plur	5	nmod:appos	O	_
+16	zonas	zona	NOUN	_	Gender=Fem|Number=Plur	2	nmod:appos	O	_
 17	com	com	ADP	_	_	18	case	O	_
 18	perdas	perda	NOUN	_	Gender=Fem|Number=Plur	16	nmod	O	_
 19	de	de	ADP	_	_	20	case	O	_
@@ -45304,16 +45304,16 @@
 30	origem	origem	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 31	;	;	PUNCT	_	_	33	punct	O	_
 32	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	33	cop	O	_
-33	comuns	comum	ADJ	_	Gender=Masc|Number=Plur	4	conj	O	_
+33	comuns	comum	ADJ	_	Gender=Masc|Number=Plur	4	parataxis	O	_
 34	,	,	PUNCT	_	_	33	punct	O	_
 35	para	para	ADP	_	_	36	case	O	_
 36	argilas	argila	NOUN	_	Gender=Fem|Number=Plur	33	obl	B=LITOLOGIA	_
 37	ou	ou	CCONJ	_	_	38	cc	O	_
 38	caulins	caulin	NOUN	_	Gender=Masc|Number=Plur	36	conj	O	_
 39	ingleses	inglês	ADJ	_	Gender=Masc|Number=Plur	38	amod	O	_
-40	nomes	nome	NOUN	_	Gender=Masc|Number=Plur	36	conj	O	_
+40	nomes	nome	NOUN	_	Gender=Masc|Number=Plur	33	nsubj	O	_
 41	como	como	ADP	_	_	42	case	O	_
-42	os	o	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	40	nmod:appos	O	_
+42	os	o	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	40	nmod	O	_
 43	de	de	ADP	_	_	44	case	O	_
 44	argilas	argila	NOUN	_	Gender=Fem|Number=Plur	42	nmod	B=LITOLOGIA	_
 45	de	de	ADP	_	_	46	case	O	_
@@ -45516,7 +45516,7 @@
 26	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	24	fixed	O	_
 27	,	,	PUNCT	_	_	29	punct	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
-29	difração	difração	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
+29	difração	difração	NOUN	_	Gender=Fem|Number=Sing	11	nmod:appos	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	raio	raio	NOUN	_	Gender=Masc|Number=Sing	29	nmod	O	_
 32	X	x	ADJ	_	Gender=Masc|Number=Sing	31	compound	O	_
@@ -47329,7 +47329,7 @@
 42	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	45	case	O	_
 43	como	como	ADP	_	_	42	fixed	O	_
 44	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	45	det	O	_
-45	plasticidade	plasticidade	NOUN	_	Gender=Fem|Number=Sing	37	nmod:appos	O	_
+45	plasticidade	plasticidade	NOUN	_	Gender=Fem|Number=Sing	34	nmod:appos	O	_
 46	,	,	PUNCT	_	_	48	punct	O	_
 47	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	48	det	O	_
 48	tixotropia	tixotropia	NOUN	_	Gender=Fem|Number=Sing	45	conj	O	_
@@ -50887,7 +50887,7 @@
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
 12	argila	argila	NOUN	_	Gender=Fem|Number=Sing	7	nmod	B=LITOLOGIA	_
 13	(	(	PUNCT	_	_	14	punct	O	_
-14	umidade	umidade	NOUN	_	Gender=Fem|Number=Sing	12	nmod:appos	O	_
+14	umidade	umidade	NOUN	_	Gender=Fem|Number=Sing	7	appos	O	_
 15	)	)	PUNCT	_	_	14	punct	O	_
 16	.	.	PUNCT	_	_	1	punct	O	_
 
@@ -63944,7 +63944,7 @@
 20	torno	torno	NOUN	_	Gender=Masc|Number=Sing	19	fixed	O	_
 21	de	de	ADP	_	_	19	fixed	O	_
 22	30	30	NUM	_	NumType=Card	23	nummod	O	_
-23	m	m	NOUN	_	Gender=Masc|Number=Plur	16	nmod:appos	O	_
+23	m	m	NOUN	_	Gender=Masc|Number=Plur	13	nmod:appos	O	_
 24	)	)	PUNCT	_	_	23	punct	O	_
 25	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -66737,7 +66737,7 @@
 9	grande	grande	ADJ	_	Gender=Masc|Number=Sing	7	obl	O	_
 10	porte	porte	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 11	(	(	PUNCT	_	_	12	punct	O	_
-12	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	10	nmod:appos	O	_
+12	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	4	nmod:appos	O	_
 13	St	St	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 14	)	)	PUNCT	_	_	12	punct	O	_
 15	,	,	PUNCT	_	_	1	punct	O	_
@@ -66977,12 +66977,12 @@
 10-11	do	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	Google	Google	PROPN	_	Gender=Masc|Number=Sing	7	conj	O	_
+12	Google	Google	PROPN	_	Gender=Masc|Number=Sing	8	conj	O	_
 13	Earth	Earth	PROPN	_	Number=Sing	12	flat:name	O	_
 14	,	,	PUNCT	_	_	17	punct	O	_
-15	como	como	ADP	_	_	17	case	O	_
+15	como	como	CCONJ	_	_	17	cc	O	_
 16	em	em	ADP	_	_	17	case	O	_
-17	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	nmod:appos	O	_
+17	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	conj	O	_
 18	,	,	PUNCT	_	_	20	punct	O	_
 19	estando	estar	AUX	_	VerbForm=Ger	20	aux	O	_
 20	ilustrada	ilustrar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	4	advcl	O	_
@@ -67846,7 +67846,7 @@
 16	,	,	PUNCT	_	_	19	punct	O	_
 17	como	como	ADP	_	_	19	case	O	_
 18	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	19	det	O	_
-19	sabkas	sabka	NOUN	_	Gender=Fem|Number=Plur	12	nmod:appos	O	_
+19	sabkas	sabka	NOUN	_	Gender=Fem|Number=Plur	14	nmod:appos	O	_
 20	(	(	PUNCT	_	_	21	punct	O	_
 21	Kocurek	Kocurek	PROPN	_	Gender=Masc|Number=Sing	8	parataxis	O	_
 22	e	e	CCONJ	_	_	21	flat:name	O	_
@@ -70273,7 +70273,7 @@
 23	(	(	PUNCT	_	_	26	punct	O	_
 24	como	como	ADP	_	_	26	case	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
-26	gás	gás	NOUN	_	Gender=Masc|Number=Sing	20	nmod:appos	B=TIPO_FLUIDO	_
+26	gás	gás	NOUN	_	Gender=Masc|Number=Sing	22	nmod:appos	B=TIPO_FLUIDO	_
 27	natural	natural	ADJ	_	Gender=Masc|Number=Sing	26	compound	O	_
 28	,	,	PUNCT	_	_	30	punct	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
@@ -78316,9 +78316,9 @@
 12	usina	usina	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 13	integrada	integrar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	12	acl	O	_
 14	,	,	PUNCT	_	_	17	punct	O	_
-15	como	como	ADV	_	_	17	cc	O	_
+15	como	como	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
-17	coqueria	coqueria	NOUN	_	Gender=Fem|Number=Sing	12	nmod:appos	O	_
+17	coqueria	coqueria	NOUN	_	Gender=Fem|Number=Sing	3	nmod:appos	O	_
 18	,	,	PUNCT	_	_	20	punct	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
 20	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
@@ -82814,12 +82814,12 @@
 41	tanto	tanto	ADV	_	_	44	advmod	O	_
 42	para	para	ADP	_	_	44	case	O	_
 43	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	44	det	O	_
-44	carbodutos	carboduto	NOUN	_	Gender=Masc|Number=Plur	35	nmod	O	_
+44	carbodutos	carboduto	NOUN	_	Gender=Masc|Number=Plur	24	obl:arg	O	_
 45	onshore	onshore	NOUN	_	Gender=Masc|Number=Sing	44	nmod	O	_
 46	como	como	ADP	_	_	49	case	O	_
 47	para	para	ADP	_	_	49	case	O	_
 48	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	49	det	O	_
-49	offshore	offshore	NOUN	_	Gender=Masc|Number=Plur	44	nmod:appos	O	_
+49	offshore	offshore	NOUN	_	Gender=Masc|Number=Plur	44	conj	O	_
 50	.	.	PUNCT	_	_	14	punct	O	_
 
 # text = Os valores máximos e mínimos foram baseados em valores reportados de carbodutos construídos ao redor do mundo, principalmente nos Estados Unidos da América.
@@ -84762,7 +84762,7 @@
 5	de	de	ADP	_	_	6	case	O	_
 6	água	água	NOUN	_	Gender=Fem|Number=Sing	4	nmod	B=TIPO_FLUIDO	_
 7	como	como	ADP	_	_	8	case	O	_
-8	método	método	NOUN	_	Gender=Masc|Number=Sing	4	nmod:appos	O	_
+8	método	método	NOUN	_	Gender=Masc|Number=Sing	1	xcomp	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 11	suplementar	suplementar	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
@@ -85138,7 +85138,7 @@
 5	de	de	ADP	_	_	6	case	O	_
 6	água	água	NOUN	_	Gender=Fem|Number=Sing	4	nmod	B=TIPO_FLUIDO	_
 7	como	como	ADP	_	_	8	case	O	_
-8	método	método	NOUN	_	Gender=Masc|Number=Sing	4	nmod:appos	O	_
+8	método	método	NOUN	_	Gender=Masc|Number=Sing	1	xcomp	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 11	suplementar	suplementar	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
@@ -85809,7 +85809,7 @@
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
 32	função	função	NOUN	_	Gender=Fem|Number=Sing	29	obl:agent	O	_
 33	gráfica	gráfico	ADJ	_	Gender=Fem|Number=Sing	32	amod	O	_
-34	a	a	ADP	_	_	29	obl	O	MWEPOS=ADV
+34	a	a	ADP	_	_	32	amod	O	MWEPOS=ADJ
 35	seguir	seguir	VERB	_	VerbForm=Inf	34	fixed	O	_
 36	:	:	PUNCT	_	_	14	punct	O	_
 37	Tanto	tanto	ADV	_	_	40	advmod	O	_
@@ -85825,7 +85825,7 @@
 47	de	de	ADP	_	_	48	case	O	_
 48	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	46	nmod	B=CAMPO	_
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	50	det	O	_
-50	valor	valor	NOUN	_	Gender=Masc|Number=Sing	32	nmod:appos	O	_
+50	valor	valor	NOUN	_	Gender=Masc|Number=Sing	62	nsubj	O	_
 51-52	da	_	_	_	_	_	_	_	_
 51	de	de	ADP	_	_	53	case	O	_
 52	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	53	det	O	_
@@ -85839,7 +85839,7 @@
 59	1000	1000	NUM	_	NumType=Card	62	nummod	O	_
 60	e	e	CCONJ	_	_	61	cc	O	_
 61	2500	2500	NUM	_	NumType=Card	59	conj	O	_
-62	mD	md	NOUN	_	Gender=Masc|Number=Plur	50	acl:relcl	O	_
+62	mD	md	NOUN	_	Gender=Masc|Number=Plur	14	parataxis	O	_
 63	(	(	PUNCT	_	_	64	punct	O	_
 64	Bruhn	Bruhn	PROPN	_	Gender=Masc|Number=Sing	14	parataxis	O	_
 65	et.	et.	PROPN	_	Number=Sing	64	flat:name	O	_
@@ -86748,7 +86748,7 @@
 29	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	30	aux:pass	O	_
 30	considerado	considerar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	O	_
-32	valor	valor	NOUN	_	Gender=Masc|Number=Sing	30	nsubj:pass	O	_
+32	valor	valor	NOUN	_	Gender=Masc|Number=Sing	30	obj	O	_
 33	máximo	grande	ADJ	_	Gender=Masc|Number=Sing	32	amod	O	_
 34	de	de	ADP	_	_	35	case	O	_
 35	injeção	injeção	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
@@ -86758,15 +86758,15 @@
 39	poço	poço	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 40	,	,	PUNCT	_	_	42	punct	O	_
 41	3.894	3.894	NUM	_	NumType=Card	42	nummod	O	_
-42	tCO2	tco2	NOUN	_	Gender=Masc|Number=Plur	32	conj	O	_
+42	tCO2	tco2	NOUN	_	Gender=Masc|Number=Plur	30	nsubj:pass	O	_
 43	/	/	ADP	_	_	44	case	O	_
 44	dia	dia	NOUN	_	Gender=Masc|Number=Sing	42	nmod	O	_
 45	tanto	tanto	ADV	_	_	42	advmod	O	_
 46	para	para	ADP	_	_	47	case	O	_
 47	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	42	nmod	B=CAMPO	_
-48	como	como	ADP	_	_	50	case	O	_
+48	como	como	CCONJ	_	_	50	cc	O	_
 49	para	para	ADP	_	_	50	case	O	_
-50	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	47	nmod:appos	B=CAMPO	_
+50	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	47	conj	B=CAMPO	_
 51	,	,	PUNCT	_	_	54	punct	O	_
 52	sendo	ser	AUX	_	VerbForm=Ger	54	cop	O	_
 53	10	10	NUM	_	NumType=Card	54	nummod	O	_
@@ -89065,9 +89065,9 @@
 9	,	,	PUNCT	_	_	12	punct	O	_
 10	como	como	ADP	_	_	14	case	O	_
 11	por	por	ADP	_	_	14	advmod	O	_
-12	exemplos	exemplo	NOUN	_	Gender=Masc|Number=Plur	8	nmod:appos	O	_
+12	exemplos	exemplo	NOUN	_	Gender=Masc|Number=Plur	11	fixed	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
-14	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
+14	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	6	nmod:appos	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	14	nmod	B=TIPO_FLUIDO	_
 17	,	,	PUNCT	_	_	18	punct	O	_
@@ -90475,7 +90475,7 @@
 61	)	)	PUNCT	_	_	58	punct	O	_
 62	,	,	PUNCT	_	_	64	punct	O	_
 63	como	como	ADP	_	_	64	case	O	_
-64	forma	forma	NOUN	_	Gender=Fem|Number=Sing	51	nmod:appos	O	_
+64	forma	forma	NOUN	_	Gender=Fem|Number=Sing	40	obl	O	_
 65	de	de	ADP	_	_	66	mark	O	_
 66	cumprir	cumprir	VERB	_	VerbForm=Inf	64	acl	O	_
 67	parte	parte	NOUN	_	Gender=Fem|Number=Sing	66	obj	O	_
@@ -91081,7 +91081,7 @@
 4	de	de	ADP	_	_	5	case	O	_
 5	Base	Base	PROPN	_	Number=Sing	3	nmod	O	_
 6	–	–	PUNCT	_	_	7	punct	O	_
-7	Emissões	emissão	NOUN	_	Gender=Fem|Number=Sing	3	nmod:appos	O	_
+7	Emissões	emissão	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
 8-9	da	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
@@ -92236,7 +92236,7 @@
 8	de	de	ADP	_	_	9	case	O	_
 9	CCGS	CCGS	PROPN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 10	como	como	ADP	_	_	11	case	O	_
-11	MDL	MDL	PROPN	_	Gender=Masc|Number=Sing	4	nmod:appos	O	_
+11	MDL	MDL	PROPN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 
 # text = Até a presente data, não há uma metodologia de cálculo de linha de base e adicionalidade pronta para ser empregada em projetos de CCGS.
 # sent_id = 161-20150810-MONOGRAFIA_0-788
@@ -103176,7 +103176,7 @@
 21	de	de	ADP	_	_	20	fixed	O	_
 22	impurezas	impureza	NOUN	_	Gender=Fem|Number=Plur	14	conj	O	_
 23	como	como	ADP	_	_	24	case	O	_
-24	areia	areia	NOUN	_	Gender=Fem|Number=Sing	22	nmod	B=LITOLOGIA	_
+24	areia	areia	NOUN	_	Gender=Fem|Number=Sing	22	nmod:appos	B=LITOLOGIA	_
 25	.	.	PUNCT	_	_	11	punct	O	_
 
 # text = Essa mistura trifásica precisa ser separada, sendo este o objetivo dos primeiros tratamentos sofrido por esta corrente.
@@ -104065,7 +104065,7 @@
 32	de	de	ADP	_	_	33	case	O	_
 33	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	20	nmod	O	_
 34	como	como	ADP	_	_	35	case	O	_
-35	água	água	NOUN	_	Gender=Fem|Number=Sing	33	nmod	B=TIPO_FLUIDO	_
+35	água	água	NOUN	_	Gender=Fem|Number=Sing	33	nmod:appos	B=TIPO_FLUIDO	_
 36	,	,	PUNCT	_	_	37	punct	O	_
 37	dióxido	dióxido	NOUN	_	Gender=Masc|Number=Sing	35	conj	O	_
 38	de	de	ADP	_	_	39	case	O	_
@@ -105637,7 +105637,7 @@
 15	de	de	ADP	_	_	16	case	O	_
 16	equipamentos	equipamento	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 17	como	como	ADP	_	_	18	case	O	_
-18	separadores	separador	NOUN	_	Gender=Masc|Number=Plur	14	nmod:appos	O	_
+18	separadores	separador	NOUN	_	Gender=Masc|Number=Plur	16	nmod:appos	O	_
 19	trifásicos	trifásico	ADJ	_	Gender=Masc|Number=Plur	18	amod	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	bombas	bomba	NOUN	_	Gender=Fem|Number=Plur	18	conj	O	_
@@ -105839,7 +105839,7 @@
 37	apoio	apoio	NOUN	_	Gender=Masc|Number=Sing	35	nmod	O	_
 38	,	,	PUNCT	_	_	40	punct	O	_
 39	como	como	ADP	_	_	40	case	O	_
-40	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	37	nmod:appos	O	_
+40	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	35	nmod:appos	O	_
 41	de	de	ADP	_	_	42	case	O	_
 42	alívio	alívio	NOUN	_	Gender=Masc|Number=Sing	40	nmod	O	_
 43	,	,	PUNCT	_	_	44	punct	O	_
@@ -109940,7 +109940,7 @@
 27	injeção	injeção	NOUN	_	Gender=Fem|Number=Sing	25	nsubj	O	_
 28	,	,	PUNCT	_	_	30	punct	O	_
 29	como	como	ADP	_	_	30	case	O	_
-30	inibidor	inibidor	NOUN	_	Gender=Masc|Number=Sing	27	nmod:appos	O	_
+30	inibidor	inibidor	NOUN	_	Gender=Masc|Number=Sing	25	obl	O	_
 31	de	de	ADP	_	_	32	case	O	_
 32	hidratos	hidrato	NOUN	_	Gender=Masc|Number=Plur	30	nmod	O	_
 33	,	,	PUNCT	_	_	36	punct	O	_
@@ -110956,15 +110956,15 @@
 
 # text = Porém como são usados em grandes quantidades, inconvenientes podem ser gerados, como por exemplo, a precipitação de sais inorgânicos.
 # sent_id = 334-20150521-MONOGRAFIA_0-420
-1	Porém	porém	ADV	_	_	4	advmod	O	_
-2	como	como	ADV	_	_	1	fixed	O	_
+1	Porém	porém	CCONJ	_	_	4	cc	O	_
+2	como	como	SCONJ	_	_	4	mark	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	usados	usar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	10	advcl	O	_
 5	em	em	ADP	_	_	7	case	O	_
 6	grandes	grande	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
 7	quantidades	quantidade	NOUN	_	Gender=Fem|Number=Plur	4	obl	O	_
 8	,	,	PUNCT	_	_	4	punct	O	_
-9	inconvenientes	inconveniente	NOUN	_	Gender=Masc|Number=Plur	10	nsubj	O	_
+9	inconvenientes	inconveniente	NOUN	_	Gender=Masc|Number=Plur	10	nsubj:pass	O	_
 10	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 11	ser	ser	AUX	_	VerbForm=Inf	12	aux:pass	O	_
 12	gerados	gerar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	10	xcomp	O	_
@@ -110974,7 +110974,7 @@
 16	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	15	fixed	O	_
 17	,	,	PUNCT	_	_	12	punct	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	precipitação	precipitação	NOUN	_	Gender=Fem|Number=Sing	12	obj	O	_
+19	precipitação	precipitação	NOUN	_	Gender=Fem|Number=Sing	9	nmod:appos	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	sais	sal	NOUN	_	Gender=Masc|Number=Plur	19	nmod	B=LITOLOGIA	_
 22	inorgânicos	inorgânico	ADJ	_	Gender=Masc|Number=Plur	21	amod	O	_
@@ -114150,7 +114150,7 @@
 # text = Os gases com maior permeância, como CO2, H2, H2O e H2S, permeiam rapidamente através da membrana (permeado) enquanto que os hidrocarbonetos, de baixa permeância, fluem pelo casco até a saída (retentado) (Figura 43).
 # sent_id = 334-20150521-MONOGRAFIA_0-521
 1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
-2	gases	gase	NOUN	_	Gender=Masc|Number=Plur	16	nsubj	O	_
+2	gases	gás	NOUN	_	Gender=Masc|Number=Plur	16	nsubj	O	_
 3	com	com	ADP	_	_	5	case	O	_
 4	maior	grande	ADJ	_	Gender=Fem|Number=Sing	5	amod	O	_
 5	permeância	permeância	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -123446,10 +123446,10 @@
 8	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 9	permite	permitir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	prever	prever	VERB	_	VerbForm=Inf	9	xcomp	O	_
-11	como	como	ADV	_	_	13	advmod	O	_
+11	como	como	SCONJ	_	_	14	mark	O	_
 12	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	13	det	O	_
-13	tensoativos	tensoativo	NOUN	_	Gender=Masc|Number=Plur	10	obj	O	_
-14	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	ccomp	O	_
+13	tensoativos	tensoativo	NOUN	_	Gender=Masc|Number=Plur	14	nsubj	O	_
+14	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	10	ccomp	O	_
 15	funcionar	funcionar	VERB	_	VerbForm=Inf	14	xcomp	O	_
 16	em	em	ADP	_	_	18	case	O	_
 17	determinados	determinado	DET	_	Gender=Masc|Number=Plur|PronType=Ind	18	det	O	_
@@ -127111,7 +127111,7 @@
 7	menores	pequeno	ADJ	_	Gender=Fem|Number=Plur	6	amod	O	_
 8	como	como	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	nitrogênio	nitrogênio	NOUN	_	Gender=Masc|Number=Sing	6	nmod:appos	O	_
+10	nitrogênio	nitrogênio	NOUN	_	Gender=Masc|Number=Sing	2	nmod:appos	O	_
 11	(	(	PUNCT	_	_	12	punct	O	_
 12	0,11	0,11	NUM	_	NumType=Card	10	appos	O	_
 13	a	a	ADP	_	_	15	case	O	_
@@ -127724,17 +127724,17 @@
 11	,	,	PUNCT	_	_	14	punct	O	_
 12	com	com	ADP	_	_	14	case	O	_
 13	alta	alto	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
-14	relação	relação	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
+14	relação	relação	NOUN	_	Gender=Fem|Number=Sing	9	conj	O	_
 15	carbono	carbono	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 16	/	/	PUNCT	_	_	17	punct	O	_
 17	hidrogênio	hidrogênio	NOUN	_	Gender=Masc|Number=Sing	15	conj	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
-19	presença	presença	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
+19	presença	presença	NOUN	_	Gender=Fem|Number=Sing	9	conj	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	heterocompostos	heterocomposto	NOUN	_	Gender=Masc|Number=Plur	19	nmod	O	_
 22	,	,	PUNCT	_	_	24	punct	O	_
 23	como	como	ADP	_	_	24	case	O	_
-24	enxofre	enxofre	NOUN	_	Gender=Masc|Number=Sing	14	nmod:appos	O	_
+24	enxofre	enxofre	NOUN	_	Gender=Masc|Number=Sing	21	nmod:appos	O	_
 25	,	,	PUNCT	_	_	26	punct	O	_
 26	oxigênio	oxigênio	NOUN	_	Gender=Masc|Number=Sing	24	conj	O	_
 27	e	e	CCONJ	_	_	28	cc	O	_
@@ -127875,7 +127875,7 @@
 5	,	,	PUNCT	_	_	2	punct	O	_
 6	como	como	ADP	_	_	8	case	O	_
 7	:	:	PUNCT	_	_	8	punct	O	_
-8	formação	formação	NOUN	_	Gender=Fem|Number=Sing	4	appos	O	_
+8	formação	formação	NOUN	_	Gender=Fem|Number=Sing	4	nmod:appos	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	depósitos	depósito	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 11	orgânicos	orgânico	ADJ	_	Gender=Masc|Number=Plur	10	amod	O	_
@@ -130996,7 +130996,7 @@
 25	transporte	transporte	NOUN	_	Gender=Masc|Number=Sing	23	nmod	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	25	nmod	B=TIPO_FLUIDO	_
-28	pesados	pesado	ADJ	_	Gender=Masc|Number=Plur|Voice=Pass	27	amod	O	_
+28	pesados	pesado	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
 29-30	pela	_	_	_	_	_	_	_	_
 29	por	por	ADP	_	_	31	case	O	_
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
@@ -131007,7 +131007,7 @@
 35	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	38	cc	O	_
 36	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	35	fixed	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
-38	escolha	escolha	NOUN	_	Gender=Fem|Number=Sing	31	nmod:appos	O	_
+38	escolha	escolha	NOUN	_	Gender=Fem|Number=Sing	23	conj	O	_
 39-40	do	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
@@ -135734,7 +135734,7 @@
 7	água	água	NOUN	_	Gender=Fem|Number=Sing	4	nmod	B=TIPO_FLUIDO	_
 8	como	como	ADP	_	_	10	case	O	_
 9	999	999	NUM	_	NumType=Card	10	nummod	O	_
-10	Kg/m3	Kg/m3	PROPN	_	Gender=Masc|Number=Sing	4	nmod:appos	O	_
+10	Kg/m3	Kg/m3	PROPN	_	Gender=Masc|Number=Sing	1	xcomp	O	_
 11	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = O rotâmentro com faixa de operação de 0,04 a 5 GPM apresentou um erro estatístico de ±0,0233 GPM e um erro sistemático de ±0,2369 GPM .
@@ -140554,7 +140554,7 @@
 25	,	,	PUNCT	_	_	28	punct	O	_
 26	como	como	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	gasolina	gasolina	NOUN	_	Gender=Fem|Number=Sing	17	nmod:appos	O	_
+28	gasolina	gasolina	NOUN	_	Gender=Fem|Number=Sing	22	nmod:appos	O	_
 29	,	,	PUNCT	_	_	31	punct	O	_
 30	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	31	det	O	_
 31	querosene	querosene	NOUN	_	Gender=Masc|Number=Sing	28	conj	O	_
@@ -140652,7 +140652,7 @@
 65	,	,	PUNCT	_	_	68	punct	O	_
 66	como	como	ADP	_	_	68	case	O	_
 67	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	68	det	O	_
-68	biocombustíveis	biocombustível	NOUN	_	Gender=Masc|Number=Plur	63	nmod:appos	O	_
+68	biocombustíveis	biocombustível	NOUN	_	Gender=Masc|Number=Plur	58	nmod:appos	O	_
 69	.	.	PUNCT	_	_	41	punct	O	_
 
 # text = Conforme Lei n°. 9478/97, biocombustível é o combustível derivado de biomassa renovável, para uso em motores a combustão interna ou conforme regulamento, para outro tipo de geração de energia, que possa substituir, parcial ou totalmente, combustíveis de origem fóssil.
@@ -141644,7 +141644,7 @@
 13	combustíveis	combustível	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
 14	renováveis	renovável	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
 15	como	como	ADP	_	_	16	case	O	_
-16	alternativa	alternativa	NOUN	_	Gender=Fem|Number=Sing	11	nmod:appos	O	_
+16	alternativa	alternativa	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 17-18	aos	_	_	_	_	_	_	_	_
 17	a	a	ADP	_	_	19	case	O	_
 18	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	19	det	O	_
@@ -143468,7 +143468,7 @@
 26	sendo	ser	AUX	_	VerbForm=Ger	30	aux	O	_
 27	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	29	det	O	_
 28	dois	dois	NUM	_	NumType=Card	29	nummod	O	_
-29	últimos	último	ADJ	_	Gender=Masc|Number=Plur|NumType=Ord	30	amod	O	_
+29	últimos	último	ADJ	_	Gender=Masc|Number=Plur|NumType=Ord	30	nsubj	O	_
 30	medidos	medir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	4	advcl	O	_
 31	em	em	ADP	_	_	32	case	O	_
 32	partes	parte	NOUN	_	Gender=Fem|Number=Plur	30	obl	O	_
@@ -143480,9 +143480,9 @@
 38	,	,	PUNCT	_	_	41	punct	O	_
 39	e	e	CCONJ	_	_	41	cc	O	_
 40	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	41	det	O	_
-41	demais	demais	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	11	conj	O	_
+41	demais	demais	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	30	conj	O	_
 42	em	em	ADP	_	_	43	case	O	_
-43	percentagens	percentagem	NOUN	_	Gender=Fem|Number=Plur	41	nmod	O	_
+43	percentagens	percentagem	NOUN	_	Gender=Fem|Number=Plur	41	orphan	O	_
 44-45	do	_	_	_	_	_	_	_	_
 44	de	de	ADP	_	_	46	case	O	_
 45	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	46	det	O	_
@@ -143490,7 +143490,7 @@
 47	(	(	PUNCT	_	_	50	punct	O	_
 48	sem	sem	ADP	_	_	50	case	O	_
 49	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	50	det	O	_
-50	água	água	NOUN	_	Gender=Fem|Number=Sing	46	nmod:appos	B=TIPO_FLUIDO	_
+50	água	água	NOUN	_	Gender=Fem|Number=Sing	46	nmod	B=TIPO_FLUIDO	_
 51	)	)	PUNCT	_	_	50	punct	O	_
 52	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -146004,7 +146004,7 @@
 2	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	15	nsubj	O	_
 3	de	de	ADP	_	_	5	case	O	_
 4	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	5	det	O	_
-5	gases	gase	NOUN	_	Gender=Masc|Number=Plur	2	nmod	O	_
+5	gases	gás	NOUN	_	Gender=Masc|Number=Plur	2	nmod	O	_
 6	como	como	ADP	_	_	8	case	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
 8	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	5	nmod:appos	O	_
@@ -146065,7 +146065,7 @@
 58	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	59	det	O	_
 59	razões	razão	NOUN	_	Gender=Fem|Number=Plur	56	nmod	O	_
 60	ar-combustível	ar-combustível	NOUN	_	Gender=Fem|Number=Sing	59	nmod	O	_
-61	real	real	ADJ	_	Gender=Fem|Number=Sing	59	amod	O	_
+61	real	real	ADJ	_	Gender=Fem|Number=Sing	60	amod	O	_
 62	e	e	CCONJ	_	_	63	cc	O	_
 63	ar-combustível	ar-combustível	NOUN	_	Gender=Fem|Number=Sing	59	conj	O	_
 64	estequiométrica	estequiométrico	ADJ	_	Gender=Fem|Number=Sing	63	amod	O	_
@@ -148337,17 +148337,17 @@
 
 # text = Várias são as matérias primas para produção de biodiesel como sementes e frutos que contêm óleo, sebo, algas, microalgas e cianobactérias, além de óleos e gorduras residuais.
 # sent_id = 271-20150810-MONOGRAFIA_0-49
-1	Várias	várias	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	4	nsubj	O	_
-2	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	cop	O	_
+1	Várias	várias	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	0	root	O	_
+2	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	1	cop	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
-4	matérias	matéria	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
+4	matérias	matéria	NOUN	_	Gender=Fem|Number=Plur	1	nsubj	O	_
 5	primas	prima	NOUN	_	Gender=Fem|Number=Plur	4	compound	O	_
 6	para	para	ADP	_	_	7	case	O	_
 7	produção	produção	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 10	como	como	ADP	_	_	11	case	O	_
-11	sementes	semente	NOUN	_	Gender=Masc|Number=Plur	7	nmod:appos	O	_
+11	sementes	semente	NOUN	_	Gender=Masc|Number=Plur	4	nmod:appos	O	_
 12	e	e	CCONJ	_	_	13	cc	O	_
 13	frutos	fruto	NOUN	_	Gender=Masc|Number=Plur	11	conj	O	_
 14	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	15	nsubj	O	_
@@ -148368,7 +148368,7 @@
 29	e	e	CCONJ	_	_	30	cc	O	_
 30	gorduras	gordura	NOUN	_	Gender=Fem|Number=Plur	28	conj	O	_
 31	residuais	residual	ADJ	_	Gender=Fem|Number=Plur	30	amod	O	_
-32	.	.	PUNCT	_	_	4	punct	O	_
+32	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = Normalmente, um triglicerídeo contém diferentes tipos de ácidos graxos, que indicam a composição do óleo vegetal.
 # sent_id = 271-20150810-MONOGRAFIA_0-50
@@ -148467,7 +148467,7 @@
 
 # text = Apesar desta abundância de fontes, apenas algumas têm importância econômica global, como a soja, o girassol e a palma.
 # sent_id = 271-20150810-MONOGRAFIA_0-53
-1	Apesar	apesar	ADV	_	_	4	case	O	_
+1	Apesar	apesar	ADV	_	_	4	case	O	MWEPOS=ADP
 2-3	desta	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	4	det	O	_
@@ -148484,7 +148484,7 @@
 14	,	,	PUNCT	_	_	17	punct	O	_
 15	como	como	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
-17	soja	soja	NOUN	_	Gender=Fem|Number=Sing	11	nmod:appos	O	_
+17	soja	soja	NOUN	_	Gender=Fem|Number=Sing	9	nmod:appos	O	_
 18	,	,	PUNCT	_	_	20	punct	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
 20	girassol	girassol	NOUN	_	Gender=Masc|Number=Sing	17	conj	O	_
@@ -152738,7 +152738,7 @@
 15	por	por	ADP	_	_	16	case	O	_
 16	chama	chama	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
 17	(	(	PUNCT	_	_	18	punct	O	_
-18	CG-FID	CG-FID	PROPN	_	Gender=Masc|Number=Sing	16	nmod:appos	O	_
+18	CG-FID	CG-FID	PROPN	_	Gender=Masc|Number=Sing	9	appos	O	_
 19	)	)	PUNCT	_	_	18	punct	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	envolvendo	envolver	VERB	_	VerbForm=Ger	7	advcl	O	_
@@ -154672,7 +154672,7 @@
 34	cálculos	cálculo	NOUN	_	Gender=Masc|Number=Plur	10	nsubj	O	_
 35	como	como	ADP	_	_	37	case	O	_
 36	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	37	det	O	_
-37	custos	custo	NOUN	_	Gender=Masc|Number=Plur	34	nmod	O	_
+37	custos	custo	NOUN	_	Gender=Masc|Number=Plur	34	nmod:appos	O	_
 38	de	de	ADP	_	_	39	case	O	_
 39	logística	logística	NOUN	_	Gender=Fem|Number=Sing	37	nmod	O	_
 40	de	de	ADP	_	_	41	case	O	_
@@ -154721,7 +154721,7 @@
 81	coleta	coleta	NOUN	_	Gender=Fem|Number=Sing	79	nmod	O	_
 82	seletiva	seletivo	ADJ	_	Gender=Fem|Number=Sing	81	amod	O	_
 83	,	,	PUNCT	_	_	84	punct	O	_
-84	disponiveis	disponivel	ADJ	_	Gender=Fem|Number=Plur	79	conj	O	_
+84	disponiveis	disponivel	ADJ	_	Gender=Masc|Number=Plur	79	acl	O	_
 85-86	no	_	_	_	_	_	_	_	_
 85	em	em	ADP	_	_	87	case	O	_
 86	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	87	det	O	_
@@ -155022,7 +155022,7 @@
 10	outros	outro	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	4	conj	O	_
 11	,	,	PUNCT	_	_	13	punct	O	_
 12	como	como	ADP	_	_	13	case	O	_
-13	manutenção	manutenção	NOUN	_	Gender=Fem|Number=Sing	4	nmod:appos	O	_
+13	manutenção	manutenção	NOUN	_	Gender=Fem|Number=Sing	10	nmod:appos	O	_
 14	,	,	PUNCT	_	_	15	punct	O	_
 15	energia	energia	NOUN	_	Gender=Fem|Number=Sing	13	conj	O	_
 16	.	.	PUNCT	_	_	4	punct	O	_
@@ -155293,9 +155293,9 @@
 16	diferentes	diferente	DET	_	Gender=Fem|Number=Plur|PronType=Ind	17	det	O	_
 17	situações	situação	NOUN	_	Gender=Fem|Number=Plur	15	nsubj:pass	O	_
 18	,	,	PUNCT	_	_	21	punct	O	_
-19	como	como	ADV	_	_	21	cc	O	_
+19	como	como	ADP	_	_	21	case	O	_
 20	:	:	PUNCT	_	_	21	punct	O	_
-21	utilização	utilização	NOUN	_	Gender=Fem|Number=Sing	17	appos	O	_
+21	utilização	utilização	NOUN	_	Gender=Fem|Number=Sing	17	nmod:appos	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	21	nmod	B=TIPO_FLUIDO	_
 24	residual	residual	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
@@ -156649,7 +156649,7 @@
 18	teor	teor	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	metais	metal	NOUN	_	Gender=Masc|Number=Plur	18	nmod	O	_
-21	como	como	ADV	_	_	23	case	O	_
+21	como	como	ADP	_	_	23	case	O	_
 22	,	,	PUNCT	_	_	23	punct	O	_
 23	Na	Na	PROPN	_	Gender=Fem|Number=Sing	20	nmod:appos	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
@@ -158006,7 +158006,7 @@
 11	residual	residual	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
 12	,	,	PUNCT	_	_	14	punct	O	_
 13	como	como	ADP	_	_	14	case	O	_
-14	sódio	sódio	NOUN	_	Gender=Masc|Number=Sing	10	nmod:appos	O	_
+14	sódio	sódio	NOUN	_	Gender=Masc|Number=Sing	7	nmod:appos	O	_
 15	,	,	PUNCT	_	_	16	punct	O	_
 16	proveniente	proveniente	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 17-18	do	_	_	_	_	_	_	_	_
@@ -159696,7 +159696,7 @@
 10	por	por	ADP	_	_	11	case	O	_
 11	mês	mês	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	11	nmod	B=TIPO_FLUIDO	_
+13	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=TIPO_FLUIDO	_
 14	diesel	diesel	NOUN	_	Gender=Masc|Number=Sing	13	compound	O	_
 15-16	nos	_	_	_	_	_	_	_	_
 15	em	em	ADP	_	_	17	case	O	_
@@ -159709,15 +159709,15 @@
 21	universidade	universidade	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
 22	e	e	CCONJ	_	_	24	cc	O	_
 23	10.000	10.000	NUM	_	NumType=Card	24	nummod	O	_
-24	litros	litro	NOUN	_	Gender=Masc|Number=Plur	21	conj	O	_
+24	litros	litro	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 25	por	por	ADP	_	_	26	case	O	_
 26	mês	mês	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 27	para	para	ADP	_	_	28	case	O	_
-28	carros	carro	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
+28	carros	carro	NOUN	_	Gender=Masc|Number=Plur	24	nmod	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
 30	equipamentos	equipamento	NOUN	_	Gender=Masc|Number=Plur	28	conj	O	_
 31	como	como	ADP	_	_	32	case	O	_
-32	cortadores	cortador	NOUN	_	Gender=Masc|Number=Plur	28	nmod:appos	O	_
+32	cortadores	cortador	NOUN	_	Gender=Masc|Number=Plur	30	nmod:appos	O	_
 33	de	de	ADP	_	_	34	case	O	_
 34	gramas	grama	NOUN	_	Gender=Masc|Number=Plur	32	nmod	O	_
 35	movidos	mover	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	34	acl	O	_
@@ -164766,14 +164766,14 @@
 11	,	,	PUNCT	_	_	12	punct	O	_
 12-13	tornando-os	_	_	_	_	_	_	_	_
 12	tornando	tornar	VERB	_	VerbForm=Ger	4	advcl	O	_
-13	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
+13	os	ele	PRON	_	Gender=Masc|Number=Plur|PronType=Prs	12	obj	O	_
 14	bastante	bastante	ADV	_	_	15	advmod	O	_
-15	atrativos	atrativo	NOUN	_	Gender=Masc|Number=Plur	12	obj	O	_
-16	industrialmente	industrialmente	ADV	_	_	12	advmod	O	_
+15	atrativos	atrativo	ADJ	_	Gender=Masc|Number=Plur	12	xcomp	O	_
+16	industrialmente	industrialmente	ADV	_	_	15	advmod	O	_
 17	tais	tal	DET	_	Gender=Masc|Number=Plur|PronType=Dem	20	case	O	_
 18	como	como	ADP	_	_	17	fixed	O	_
 19	:	:	PUNCT	_	_	20	punct	O	_
-20	cosméticos	cosmético	NOUN	_	Gender=Masc|Number=Plur	15	nmod:appos	O	_
+20	cosméticos	cosmético	NOUN	_	Gender=Masc|Number=Plur	8	nmod:appos	O	_
 21	,	,	PUNCT	_	_	22	punct	O	_
 22	detergentes	detergente	NOUN	_	Gender=Masc|Number=Plur	20	conj	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
@@ -165365,7 +165365,7 @@
 17	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	20	case	O	_
 18	como	como	ADV	_	_	17	fixed	O	_
 19	:	:	PUNCT	_	_	20	punct	O	_
-20	flotação	flotação	NOUN	_	Gender=Fem|Number=Sing	11	appos	O	_
+20	flotação	flotação	NOUN	_	Gender=Fem|Number=Sing	15	nmod:appos	O	_
 21	,	,	PUNCT	_	_	22	punct	O	_
 22	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
@@ -167104,7 +167104,7 @@
 17	,	,	PUNCT	_	_	20	punct	O	_
 18	tais	tal	DET	_	Gender=Masc|Number=Plur|PronType=Dem	20	case	O	_
 19	como	como	ADP	_	_	18	fixed	O	_
-20	inibidores	inibidor	NOUN	_	Gender=Masc|Number=Plur	11	nmod:appos	O	_
+20	inibidores	inibidor	NOUN	_	Gender=Masc|Number=Plur	4	nmod:appos	O	_
 21	de	de	ADP	_	_	22	case	O	_
 22	corrosão	corrosão	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 23	e	e	CCONJ	_	_	24	cc	O	_
@@ -168549,9 +168549,9 @@
 25	flotação	flotação	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 26	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	27	aux:pass	O	_
 27	empregados	empregar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	5	conj	O	_
-28	xileno	xileno	NOUN	_	Gender=Masc|Number=Sing	27	xcomp	O	_
+28	xileno	xileno	NOUN	_	Gender=Masc|Number=Sing	27	nsubj:pass	O	_
 29	como	como	ADP	_	_	30	case	O	_
-30	poluente	poluente	NOUN	_	Gender=Masc|Number=Sing	28	nmod:appos	O	_
+30	poluente	poluente	NOUN	_	Gender=Masc|Number=Sing	27	xcomp	O	_
 31	modelo	modelo	NOUN	_	Gender=Masc|Number=Sing	30	nmod	O	_
 32	,	,	PUNCT	_	_	38	punct	O	_
 33	já	já	ADV	_	_	38	advmod	O	_
@@ -168579,7 +168579,7 @@
 10	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	13	case	O	_
 11	como	como	ADV	_	_	10	fixed	O	_
 12	:	:	PUNCT	_	_	13	punct	O	_
-13	redução	redução	NOUN	_	Gender=Fem|Number=Sing	8	appos	O	_
+13	redução	redução	NOUN	_	Gender=Fem|Number=Sing	8	nmod:appos	O	_
 14-15	do	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -170978,7 +170978,7 @@
 2	de	de	ADP	_	_	3	case	O	_
 3	Molibdênio	Molibdênio	PROPN	_	Gender=Masc|Number=Sing	1	nmod	O	_
 4	como	como	ADP	_	_	5	case	O	_
-5	catalisadores	catalisador	NOUN	_	Gender=Masc|Number=Plur	1	nmod:appos	O	_
+5	catalisadores	catalisador	NOUN	_	Gender=Masc|Number=Plur	1	nmod	O	_
 
 # text = 1.1.2.1. Carbeto de Molibdênio suportado em zeólitas
 # sent_id = 25-20150123-MONOGRAFIA_0-8
@@ -171855,7 +171855,7 @@
 4	de	de	ADP	_	_	5	case	O	_
 5	Molibdênio	Molibdênio	PROPN	_	Gender=Masc|Number=Sing	3	nmod	O	_
 6	como	como	ADP	_	_	7	case	O	_
-7	catalisadores	catalisador	NOUN	_	Gender=Masc|Number=Plur	3	nmod:appos	O	_
+7	catalisadores	catalisador	NOUN	_	Gender=Masc|Number=Plur	3	nmod	O	_
 
 # text = Uma das aplicações dos carbetos está na catálise heterogênea.
 # sent_id = 25-20150123-MONOGRAFIA_0-55
@@ -172096,10 +172096,10 @@
 19	e	e	CCONJ	_	_	22	cc	O	_
 20	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	22	det	O	_
 21	usuais	usual	ADJ	_	Gender=Masc|Number=Plur	22	amod	O	_
-22	materiais-suporte	materiais-suporte	NOUN	_	Gender=Masc|Number=Plur	16	conj	O	_
+22	materiais-suporte	material-suporte	NOUN	_	Gender=Masc|Number=Plur	16	conj	O	_
 23	tais	tal	DET	_	Gender=Masc|Number=Plur|PronType=Dem	25	case	O	_
 24	como	como	ADP	_	_	23	fixed	O	_
-25	sílica	sílica	NOUN	_	Gender=Fem|Number=Sing	16	nmod:appos	O	_
+25	sílica	sílica	NOUN	_	Gender=Fem|Number=Sing	22	nmod:appos	O	_
 26	ou	ou	CCONJ	_	_	27	cc	O	_
 27	alumina	alumina	NOUN	_	Gender=Fem|Number=Sing	25	conj	O	_
 28	obriga	obrigar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
@@ -172490,7 +172490,7 @@
 20	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	cop	O	_
 21	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
 22	entender	entender	VERB	_	VerbForm=Inf	21	csubj	O	_
-23	como	como	ADV	_	_	27	advmod	O	_
+23	como	como	SCONJ	_	_	27	mark	O	_
 24	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	25	det	O	_
 25	elétrons	elétron	NOUN	_	Gender=Masc|Number=Plur	27	nsubj:pass	O	_
 26	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	27	aux:pass	O	_
@@ -174316,9 +174316,9 @@
 103	de	de	ADP	_	_	104	case	O	_
 104	forma	forma	NOUN	_	Gender=Fem|Number=Sing	102	nmod	O	_
 105	,	,	PUNCT	_	_	108	punct	O	_
-106	como	como	ADV	_	_	108	cc	O	_
+106	como	como	ADP	_	_	108	case	O	_
 107	,	,	PUNCT	_	_	108	punct	O	_
-108	seletividade	seletividade	NOUN	_	Gender=Fem|Number=Sing	102	conj	O	_
+108	seletividade	seletividade	NOUN	_	Gender=Fem|Number=Sing	102	nmod:appos	O	_
 109	de	de	ADP	_	_	110	case	O	_
 110	reagente	reagente	NOUN	_	Gender=Masc|Number=Sing	108	nmod	O	_
 111	,	,	PUNCT	_	_	113	punct	O	_
@@ -174818,17 +174818,17 @@
 16	silício	silício	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 17	,	,	PUNCT	_	_	19	punct	O	_
 18	como	como	ADP	_	_	19	case	O	_
-19	razão	razão	NOUN	_	Gender=Fem|Number=Sing	13	nmod:appos	O	_
+19	razão	razão	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 20	sílica/alumina	sílica/alumino	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21	,	,	PUNCT	_	_	22	punct	O	_
 22	abreviadamente	abreviadamente	ADV	_	_	19	advmod	O	_
 23	,	,	PUNCT	_	_	22	punct	O	_
-24	SAR	SAR	PROPN	_	Gender=Masc|Number=Sing	19	conj	O	_
+24	SAR	SAR	PROPN	_	Gender=Masc|Number=Sing	19	appos	O	_
 25	,	,	PUNCT	_	_	28	punct	O	_
 26-27	do	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
-28	inglês	inglês	NOUN	_	Gender=Masc|Number=Sing	19	conj	O	_
+28	inglês	inglês	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 29	silica/	silica/	X	_	_	28	nmod	O	_
 30	alumina	alumina	X	_	_	28	nmod	O	_
 31	ratio	ratio	X	_	_	28	flat:foreign	O	_
@@ -176377,7 +176377,7 @@
 
 # text = °C e final de 500 °C por 30 min com uma taxa de 10 °C min-1 tendo o gás Hélio como gás de fluxo.
 # sent_id = 25-20150123-MONOGRAFIA_0-212
-1	°	°	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+1	°	°	NOUN	_	Gender=Masc|Number=Sing	0	root	O	sentenciação
 2	C	C	PROPN	_	Gender=Masc|Number=Sing	1	compound	O	_
 3	e	e	CCONJ	_	_	4	cc	O	_
 4	final	final	NOUN	_	Gender=Masc|Number=Sing	1	conj	O	_
@@ -176873,7 +176873,7 @@
 25	razão	razão	NOUN	_	Gender=Fem|Number=Sing	22	obj	O	_
 26	sílica/alumina	sílica/alumino	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
 27	(	(	PUNCT	_	_	28	punct	O	_
-28	SAR	SAR	PROPN	_	Gender=Masc|Number=Sing	25	nmod:appos	O	_
+28	SAR	SAR	PROPN	_	Gender=Masc|Number=Sing	25	appos	O	_
 29	)	)	PUNCT	_	_	28	punct	O	_
 30	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -185427,7 +185427,7 @@
 9	forma	forma	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
 10	impressa	imprimir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	9	acl	O	_
 11	em	em	ADP	_	_	12	case	O	_
-12	papel	papel	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
+12	papel	papel	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	anexos	anexo	NOUN	_	Gender=Masc|Number=Plur	6	parataxis	O	_
 15	1	1	NUM	_	NumType=Card	14	nummod	O	_
@@ -185443,7 +185443,7 @@
 24	forma	forma	NOUN	_	Gender=Fem|Number=Sing	9	conj	O	_
 25	digital	digital	ADJ	_	Gender=Fem|Number=Sing	24	amod	O	_
 26	como	como	ADP	_	_	27	case	O	_
-27	arquivo	arquivo	NOUN	_	Gender=Masc|Number=Sing	24	nmod:appos	O	_
+27	arquivo	arquivo	NOUN	_	Gender=Masc|Number=Sing	24	orphan	O	_
 28	empilhado	empilhar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	27	acl	O	_
 29-30	do	_	_	_	_	_	_	_	_
 29	de	de	ADP	_	_	31	case	O	_
@@ -188980,7 +188980,7 @@
 8	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	11	det	O	_
 9	como	como	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
-11	serra	serra	PROPN	_	Gender=Fem|Number=Sing	5	nmod:appos	B=CAMPO	_
+11	serra	serra	PROPN	_	Gender=Fem|Number=Sing	1	nmod:appos	B=CAMPO	_
 12-13	da	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	11	flat:name	I=CAMPO	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	flat:name	I=CAMPO	_
@@ -190986,7 +190986,7 @@
 11	contatos	contato	NOUN	_	Gender=Masc|Number=Plur	26	nsubj:pass	O	_
 12-13	da	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	15	case	O	_
-13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
+13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	suíte	suíte	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 15	alcalina	alcalina	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	com	com	ADP	_	_	18	case	O	_
@@ -191024,21 +191024,21 @@
 45	bolsões	bolsão	NOUN	_	Gender=Masc|Number=Plur	39	nmod	O	_
 46	pegmatóides	pegmatóide	ADJ	_	Gender=Masc|Number=Plur	45	amod	O	_
 47	,	,	PUNCT	_	_	51	punct	O	_
-48	como	como	ADP	_	_	51	case	O	_
+48	como	como	CCONJ	_	_	51	cc	O	_
 49-50	no	_	_	_	_	_	_	_	_
 49	em	em	ADP	_	_	51	case	O	_
 50	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	51	det	O	_
-51	Pontal	pontal	PROPN	_	Gender=Masc|Number=Sing	45	nmod:appos	O	_
+51	Pontal	pontal	PROPN	_	Gender=Masc|Number=Sing	31	conj	O	_
 52-53	do	_	_	_	_	_	_	_	_
 52	de	de	ADP	_	_	51	flat:name	O	_
 53	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	51	flat:name	O	_
 54	Atalaia	Atalaia	PROPN	_	Number=Sing	51	flat:name	O	_
 55	,	,	PUNCT	_	_	51	punct	O	_
-56	junto	junto	ADV	_	_	51	advmod	O	_
+56	junto	junto	ADV	_	_	59	cc	O	_
 57-58	às	_	_	_	_	_	_	_	_
-57	a	a	ADP	_	_	59	case	O	_
+57	a	a	ADP	_	_	56	fixed	O	_
 58	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	59	det	O	_
-59	Prainhas	Prainhas	PROPN	_	Gender=Fem|Number=Plur	56	obl	O	_
+59	Prainhas	Prainhas	PROPN	_	Gender=Fem|Number=Plur	51	conj	O	_
 60	,	,	PUNCT	_	_	56	punct	O	_
 61	onde	onde	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	62	obl	O	_
 62	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	59	acl:relcl	O	_
@@ -193036,7 +193036,7 @@
 28	potenciais	potencial	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
 29	como	como	ADP	_	_	31	case	O	_
 30	principal	principal	ADJ	_	Gender=Fem|Number=Sing	31	amod	O	_
-31	ferramenta	ferramenta	NOUN	_	Gender=Fem|Number=Sing	27	nmod:appos	O	_
+31	ferramenta	ferramenta	NOUN	_	Gender=Fem|Number=Sing	24	xcomp	O	_
 32-33	na	_	_	_	_	_	_	_	_
 32	em	em	ADP	_	_	34	case	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
@@ -199450,7 +199450,7 @@
 31	campo	campo	NOUN	_	Gender=Masc|Number=Sing	27	appos	O	_
 32	gravimétrico	gravimétrico	ADJ	_	Gender=Masc|Number=Sing	31	amod	O	_
 33	como	como	ADP	_	_	34	case	O	_
-34	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	31	nmod:appos	O	_
+34	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	31	nmod	O	_
 35	)	)	PUNCT	_	_	31	punct	O	_
 36	a	a	ADP	_	_	40	case	O	MWEPOS=ADP
 37	partir	partir	VERB	_	VerbForm=Inf	36	fixed	O	_
@@ -202692,7 +202692,7 @@
 8	introdução	introdução	NOUN	_	Gender=Fem|Number=Sing	6	nsubj:pass	O	_
 9	,	,	PUNCT	_	_	11	punct	O	_
 10	como	como	ADP	_	_	11	case	O	_
-11	informação	informação	NOUN	_	Gender=Fem|Number=Sing	8	nmod:appos	O	_
+11	informação	informação	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
 12	a	a	ADP	_	_	11	amod	O	MWEPOS=ADJ
 13	priori	priori	NOUN	_	Gender=Fem|Number=Sing	12	fixed	O	_
 14	,	,	PUNCT	_	_	11	punct	O	_
@@ -204320,7 +204320,7 @@
 
 # text = ou mesmo de fontes indiretas como outros métodos de investigação geofísica.
 # sent_id = 20-20140904-TESEDSC_0-546
-1	ou	ou	CCONJ	_	_	4	cc	O	_
+1	ou	ou	CCONJ	_	_	4	cc	O	sentenciação
 2	mesmo	mesmo	ADV	_	_	4	advmod	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	fontes	fonte	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
@@ -204755,7 +204755,7 @@
 23	reta	reta	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
 24	como	como	ADP	_	_	26	case	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
-26	elemento	elemento	NOUN	_	Gender=Masc|Number=Sing	21	nmod:appos	O	_
+26	elemento	elemento	NOUN	_	Gender=Masc|Number=Sing	19	xcomp	O	_
 27	geométrico	geométrico	ADJ	_	Gender=Masc|Number=Sing	26	amod	O	_
 28-29	da	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
@@ -206626,7 +206626,7 @@
 
 # text = Como é previsível, o método clássico da Deconvolução de Euler fornece uma estimativa de profundidade apenas do topo da falha ou contato, impossibilitando a estimativa de continuidade da mesma em profundidade.
 # sent_id = 20-20140904-TESEDSC_0-606
-1	Como	como	ADV	_	_	3	advmod	O	_
+1	Como	como	SCONJ	_	_	3	mark	O	_
 2	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	cop	O	_
 3	previsível	previsível	ADJ	_	Gender=Masc|Number=Sing	13	advcl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
@@ -207390,7 +207390,7 @@
 16	processo	processo	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 17	empírico	empírico	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
 18	,	,	PUNCT	_	_	11	punct	O	_
-19	como	como	ADV	_	_	22	cc	O	_
+19	como	como	SCONJ	_	_	33	mark	O	_
 20	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	22	det	O	_
 21	duas	dois	NUM	_	NumType=Card	22	nummod	O	_
 22	metodologias	metodologia	NOUN	_	Gender=Fem|Number=Plur	33	nsubj	O	_
@@ -207404,7 +207404,7 @@
 30	Compacta	Compacta	PROPN	_	Number=Sing	29	flat:name	O	_
 31	Generalizada	Generalizada	PROPN	_	Number=Sing	29	flat:name	O	_
 32	)	)	PUNCT	_	_	25	punct	O	_
-33	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	conj	O	_
+33	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	ccomp	O	_
 34	fornecer	fornecer	VERB	_	VerbForm=Inf	33	xcomp	O	_
 35	soluções	solução	NOUN	_	Gender=Fem|Number=Plur	34	obj	O	_
 36	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	37	nsubj	O	_
@@ -210575,11 +210575,11 @@
 23	está	estar	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	aux:pass	O	_
 24	alicerçada	alicerçar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	19	advcl	O	_
 25	por	por	ADP	_	_	26	case	O	_
-26	dados	dado	NOUN	_	Gender=Masc|Number=Plur	24	obl	O	_
+26	dados	dado	NOUN	_	Gender=Masc|Number=Plur	24	obl:agent	O	_
 27	de	de	ADP	_	_	28	case	O	_
 28	sísmica	sísmica	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 29	como	como	ADP	_	_	30	case	O	_
-30	informação	informação	NOUN	_	Gender=Fem|Number=Sing	26	nmod:appos	O	_
+30	informação	informação	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 31	a	a	ADP	_	_	32	case	O	_
 32	priori.	priori.	NOUN	_	Gender=Fem|Number=Sing	30	nmod	O	_
 33	.	.	PUNCT	_	_	19	punct	O	_
@@ -216585,7 +216585,7 @@
 20	reta	reta	NOUN	_	Gender=Fem|Number=Sing	18	nsubj:pass	O	_
 21	vertical	vertical	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 22	como	como	ADP	_	_	23	case	O	_
-23	vínculo	vínculo	NOUN	_	Gender=Masc|Number=Sing	20	nmod:appos	O	_
+23	vínculo	vínculo	NOUN	_	Gender=Masc|Number=Sing	18	xcomp	O	_
 24	para	para	ADP	_	_	25	case	O	_
 25	inversão	inversão	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 26	compacta	compacto	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
@@ -232677,7 +232677,7 @@
 41	magnéticas	magnético	ADJ	_	Gender=Fem|Number=Plur	40	amod	O	_
 42	,	,	PUNCT	_	_	44	punct	O	_
 43	como	como	ADP	_	_	44	case	O	_
-44	direções	direção	NOUN	_	Gender=Fem|Number=Plur	36	nmod:appos	O	_
+44	direções	direção	NOUN	_	Gender=Fem|Number=Plur	40	nmod:appos	O	_
 45	de	de	ADP	_	_	46	case	O	_
 46	magnetizações	magnetização	NOUN	_	Gender=Fem|Number=Plur	44	nmod	O	_
 47	e	e	CCONJ	_	_	48	cc	O	_
@@ -235481,7 +235481,7 @@
 7	,	,	PUNCT	_	_	10	punct	O	_
 8	tal	tal	ADV	_	_	10	cc	O	_
 9	como	como	ADP	_	_	8	fixed	O	_
-10	permeabilidade	permeabilidade	NOUN	_	Gender=Fem|Number=Sing	5	nmod:appos	O	_
+10	permeabilidade	permeabilidade	NOUN	_	Gender=Fem|Number=Sing	2	nmod:appos	O	_
 11	ou	ou	CCONJ	_	_	12	cc	O	_
 12	porosidade	porosidade	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
 13	,	,	PUNCT	_	_	10	punct	O	_
@@ -239020,9 +239020,9 @@
 11	de	de	ADP	_	_	13	case	O	_
 12	vários	vário	DET	_	Gender=Masc|Number=Plur|PronType=Ind	13	det	O	_
 13	fatores	fator	NOUN	_	Gender=Masc|Number=Plur	10	nmod	O	_
-14	como	como	ADV	_	_	16	cc	O	_
+14	como	como	ADP	_	_	16	case	O	_
 15	:	:	PUNCT	_	_	16	punct	O	_
-16	tipos	tipo	NOUN	_	Gender=Masc|Number=Plur	13	appos	O	_
+16	tipos	tipo	NOUN	_	Gender=Masc|Number=Plur	13	nmod:appos	O	_
 17	de	de	ADP	_	_	18	case	O	_
 18	formações	formação	NOUN	_	Gender=Fem|Number=Plur	16	nmod	O	_
 19	a	a	ADP	_	_	21	mark	O	_
@@ -239186,7 +239186,7 @@
 28	como	como	ADP	_	_	31	case	O	_
 29	:	:	PUNCT	_	_	31	punct	O	_
 30	alto	alto	ADJ	_	Gender=Masc|Number=Sing	31	amod	O	_
-31	custo	custo	NOUN	_	Gender=Masc|Number=Sing	27	appos	O	_
+31	custo	custo	NOUN	_	Gender=Masc|Number=Sing	27	nmod:appos	O	_
 32	(	(	PUNCT	_	_	40	punct	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	O	_
 34	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	40	nsubj	B=TIPO_FLUIDO	_
@@ -243566,11 +243566,11 @@
 19	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	20	aux:pass	O	_
 20	conhecida	conhecer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	9	conj	O	_
 21	como	como	ADP	_	_	23	case	O	_
-22	“	"	PUNCT	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	23	punct	O	_
-23	curva	curva	NOUN	_	Gender=Fem|Number=Sing	20	nmod:appos	O	_
+22	“	"	PUNCT	_	_	23	punct	O	_
+23	curva	curva	NOUN	_	Gender=Fem|Number=Sing	20	xcomp	O	_
 24	de	de	ADP	_	_	25	case	O	_
 25	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	23	nmod	O	_
-26	”	"	PUNCT	_	Gender=Masc|Number=Sing	23	punct	O	_
+26	”	"	PUNCT	_	_	23	punct	O	_
 27	.	.	PUNCT	_	_	9	punct	O	_
 
 # text = Uma curva de fluxo é, portanto, um registro gráfico que mostra como a tensão cisalhante varia em função da taxa de cisalhamento (Machado, 2002).
@@ -244884,7 +244884,7 @@
 7	de	de	ADP	_	_	8	case	O	_
 8	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=TIPO_FLUIDO	_
 9	como	como	ADP	_	_	10	case	O	_
-10	água	água	NOUN	_	Gender=Fem|Number=Sing	6	nmod:appos	B=TIPO_FLUIDO	_
+10	água	água	NOUN	_	Gender=Fem|Number=Sing	8	nmod:appos	B=TIPO_FLUIDO	_
 11	e	e	CCONJ	_	_	12	cc	O	_
 12	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	10	conj	B=TIPO_FLUIDO	_
 13	,	,	PUNCT	_	_	6	punct	O	_
@@ -244903,7 +244903,7 @@
 26	seja	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	29	cop	O	_
 27	em	em	ADP	_	_	29	case	O	_
 28	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
-29	capilar	capilar	NOUN	_	Gender=Masc|Number=Sing	15	conj	O	_
+29	capilar	capilar	NOUN	_	Gender=Masc|Number=Sing	15	parataxis	O	_
 30	ou	ou	CCONJ	_	_	33	cc	O	_
 31	em	em	ADP	_	_	33	case	O	_
 32	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
@@ -245566,7 +245566,7 @@
 25	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	28	case	O	_
 26	como	como	ADV	_	_	25	fixed	O	_
 27	:	:	PUNCT	_	_	28	punct	O	_
-28	peso	peso	NOUN	_	Gender=Masc|Number=Sing	18	appos	O	_
+28	peso	peso	NOUN	_	Gender=Masc|Number=Sing	23	nmod:appos	O	_
 29	molecular	molecular	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
 30	,	,	PUNCT	_	_	31	punct	O	_
 31	tamanho	tamanho	NOUN	_	Gender=Masc|Number=Sing	28	conj	O	_
@@ -245593,7 +245593,7 @@
 50	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	53	case	O	_
 51	como	como	ADV	_	_	50	fixed	O	_
 52	:	:	PUNCT	_	_	53	punct	O	_
-53	pH	ph	NOUN	_	Gender=Masc|Number=Sing	43	nmod:appos	O	_
+53	pH	ph	NOUN	_	Gender=Masc|Number=Sing	48	nmod:appos	O	_
 54	,	,	PUNCT	_	_	55	punct	O	_
 55	salinidade	salinidade	NOUN	_	Gender=Fem|Number=Sing	53	conj	O	_
 56	(	(	PUNCT	_	_	57	punct	O	_
@@ -252685,7 +252685,7 @@
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	1	obj	O	_
 5	como	como	ADP	_	_	6	case	O	_
-6	incompressível	incompressível	ADJ	_	Gender=Masc|Number=Sing	2	nmod:appos	O	_
+6	incompressível	incompressível	ADJ	_	Gender=Masc|Number=Sing	1	xcomp	O	_
 7	,	,	PUNCT	_	_	1	punct	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
 9	volumes	volume	NOUN	_	Gender=Masc|Number=Plur	12	nsubj	O	_
@@ -265594,7 +265594,7 @@
 30	)	)	PUNCT	_	_	28	punct	O	_
 31	,	,	PUNCT	_	_	33	punct	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
-33	limite	limite	NOUN	_	Gender=Masc|Number=Sing	24	conj	O	_
+33	limite	limite	NOUN	_	Gender=Masc|Number=Sing	6	conj	O	_
 34	de	de	ADP	_	_	35	case	O	_
 35	escoamento	escoamento	NOUN	_	Gender=Masc|Number=Sing	33	nmod	O	_
 36	(	(	PUNCT	_	_	37	punct	O	_
@@ -265603,7 +265603,7 @@
 39	,	,	PUNCT	_	_	42	punct	O	_
 40	como	como	ADP	_	_	42	case	O	_
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
-42	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	33	nmod:appos	O	_
+42	tensão	tensão	NOUN	_	Gender=Fem|Number=Sing	33	orphan	O	_
 43	mínima	pequeno	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
 44	(	(	PUNCT	_	_	45	punct	O	_
 45	𝜏𝑙	𝜏𝑙	NOUN	_	Gender=Fem|Number=Sing	42	nmod:appos	O	_
@@ -269565,7 +269565,7 @@
 16	destilada	destilar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	15	acl	O	_
 17	,	,	PUNCT	_	_	19	punct	O	_
 18	como	como	ADP	_	_	19	case	O	_
-19	fase	fase	NOUN	_	Gender=Fem|Number=Sing	15	nmod:appos	O	_
+19	fase	fase	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
 20	aquosa	aquoso	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21	;	;	PUNCT	_	_	23	punct	O	_
 22	/	/	PUNCT	_	_	23	punct	O	_
@@ -269573,18 +269573,18 @@
 24	NaCl	NaCl	PROPN	_	Gender=Masc|Number=Sing	23	nmod	O	_
 25	,	,	PUNCT	_	_	27	punct	O	_
 26	para	para	ADP	_	_	27	mark	O	_
-27	transformar	transformar	VERB	_	VerbForm=Inf	9	advcl	O	_
+27	transformar	transformar	VERB	_	VerbForm=Inf	24	acl	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	água	água	NOUN	_	Gender=Fem|Number=Sing	27	obj	B=TIPO_FLUIDO	_
 30	destilada	destilar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	29	acl	O	_
 31	em	em	ADP	_	_	32	case	O	_
-32	salmoura	salmoura	NOUN	_	Gender=Fem|Number=Sing	30	obl	O	_
+32	salmoura	salmoura	NOUN	_	Gender=Fem|Number=Sing	27	obl:arg	O	_
 33	;	;	PUNCT	_	_	34	punct	O	_
 34	DÁ	DÁ	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
 35			DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	flat:name	O	_
 36	n-Parafina	n-Parafina	PROPN	_	Number=Sing	34	flat:name	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
-38	representando	representar	VERB	_	VerbForm=Ger	9	advcl	O	_
+38	representando	representar	VERB	_	VerbForm=Ger	36	acl	O	_
 39	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
 40	fase	fase	NOUN	_	Gender=Fem|Number=Sing	38	obj	O	_
 41	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	40	nmod	B=TIPO_FLUIDO	_
@@ -271976,7 +271976,7 @@
 4	concentrações	concentração	NOUN	_	Gender=Fem|Number=Plur	12	obl	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	como	como	ADP	_	_	7	case	O	_
-7	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	4	nmod:appos	B=TIPO_FLUIDO	_
+7	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	12	obl	B=TIPO_FLUIDO	_
 8	pseudoplástico	pseudoplástico	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
 9	,	,	PUNCT	_	_	7	punct	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -275302,9 +275302,9 @@
 38	determinados	determinar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	25	conj	O	_
 39	por	por	ADP	_	_	40	case	O	_
 40	fatores	fator	NOUN	_	Gender=Masc|Number=Plur	38	obl:agent	O	_
-41	como	como	ADV	_	_	43	cc	O	_
+41	como	como	ADP	_	_	43	case	O	_
 42	:	:	PUNCT	_	_	43	punct	O	_
-43	composição	composição	NOUN	_	Gender=Fem|Number=Sing	40	appos	O	_
+43	composição	composição	NOUN	_	Gender=Fem|Number=Sing	40	nmod:appos	O	_
 44	química	química	ADJ	_	Gender=Fem|Number=Sing	43	amod	O	_
 45-46	do	_	_	_	_	_	_	_	_
 45	de	de	ADP	_	_	47	case	O	_
@@ -277113,7 +277113,7 @@
 7	,	,	PUNCT	_	_	10	punct	O	_
 8	como	como	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	Oxigênio	Oxigênio	PROPN	_	Gender=Masc|Number=Sing	5	nmod:appos	O	_
+10	Oxigênio	Oxigênio	PROPN	_	Gender=Masc|Number=Sing	3	nmod:appos	O	_
 11	(	(	PUNCT	_	_	12	punct	O	_
 12	O	O	PROPN	_	Gender=Masc|Number=Sing	10	appos	O	_
 13	)	)	PUNCT	_	_	12	punct	O	_
@@ -277878,7 +277878,7 @@
 35	,	,	PUNCT	_	_	38	punct	O	_
 36	como	como	ADP	_	_	38	case	O	_
 37	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	38	det	O	_
-38	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	29	nmod:appos	B=TIPO_FLUIDO	_
+38	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	34	nmod:appos	B=TIPO_FLUIDO	_
 39	pesado	pesado	ADJ	_	Gender=Masc|Number=Sing	38	amod	O	_
 40	,	,	PUNCT	_	_	39	punct	O	_
 41	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	43	obl	O	_
@@ -278090,7 +278090,7 @@
 7	por	por	ADP	_	_	6	fixed	O	_
 8	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	6	fixed	O	_
 9	,	,	PUNCT	_	_	10	punct	O	_
-10	condições	condição	NOUN	_	Gender=Fem|Number=Plur	2	conj	O	_
+10	condições	condição	NOUN	_	Gender=Fem|Number=Plur	2	nmod:appos	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
@@ -278110,7 +278110,7 @@
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	eficiência	eficiência	NOUN	_	Gender=Fem|Number=Sing	25	obj	O	_
 28	de	de	ADP	_	_	30	case	O	_
-29	cada	cada	DET	_	Gender=Masc|Number=Sing	30	det	O	_
+29	cada	cada	DET	_	Gender=Masc|Number=Sing|PronType=Ind	30	det	O	_
 30	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	27	nmod	O	_
 31-32	desses	_	_	_	_	_	_	_	_
 31	de	de	ADP	_	_	33	case	O	_
@@ -278419,7 +278419,7 @@
 26	onshore	onshore	NOUN	_	Gender=Fem|Number=Sing	25	nmod	O	_
 27	(	(	PUNCT	_	_	29	punct	O	_
 28	em	em	ADP	_	_	29	case	O	_
-29	terra	terra	NOUN	_	Gender=Fem|Number=Sing	26	nmod:appos	O	_
+29	terra	terra	NOUN	_	Gender=Fem|Number=Sing	26	appos	O	_
 30	)	)	PUNCT	_	_	29	punct	O	_
 31	,	,	PUNCT	_	_	34	punct	O	_
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
@@ -280150,9 +280150,9 @@
 34	aves	ave	NOUN	_	Gender=Fem|Number=Plur	22	obj	O	_
 35	vinda	vir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	34	acl	O	_
 36	de	de	ADP	_	_	37	case	O	_
-37	países	país	NOUN	_	Gender=Masc|Number=Plur	34	nmod	O	_
+37	países	país	NOUN	_	Gender=Masc|Number=Plur	35	obl	O	_
 38	como	como	ADP	_	_	39	case	O	_
-39	Estados	Estados	PROPN	_	Gender=Masc|Number=Plur	34	nmod:appos	O	_
+39	Estados	Estados	PROPN	_	Gender=Masc|Number=Plur	37	nmod:appos	O	_
 40	Unidos	Unidos	PROPN	_	Number=Sing	39	flat:name	O	_
 41	e	e	CCONJ	_	_	42	cc	O	_
 42	Canadá	Canadá	PROPN	_	Gender=Masc|Number=Sing	39	conj	O	_
@@ -280859,7 +280859,7 @@
 32	estuários	estuário	NOUN	_	Gender=Masc|Number=Plur	29	obl:arg	B=SISTEMA_DEPOSICIONAL	_
 33	locais	local	ADJ	_	Gender=Masc|Number=Plur	32	amod	O	_
 34	como	como	ADP	_	_	35	case	O	_
-35	criadouros	criadouro	NOUN	_	Gender=Masc|Number=Plur	32	nmod:appos	O	_
+35	criadouros	criadouro	NOUN	_	Gender=Masc|Number=Plur	29	xcomp	O	_
 36	e	e	CCONJ	_	_	38	cc	O	_
 37	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	38	det	O	_
 38	local	local	NOUN	_	Gender=Masc|Number=Sing	35	conj	O	_
@@ -280871,7 +280871,7 @@
 44	moluscos	molusco	NOUN	_	Gender=Masc|Number=Plur	42	nmod	O	_
 45	,	,	PUNCT	_	_	46	punct	O	_
 46	crustáceos	crustáceo	NOUN	_	Gender=Masc|Number=Plur	44	conj	O	_
-47	peixes	peixe	ADJ	_	Gender=Masc|Number=Plur	44	conj	O	_
+47	peixes	peixe	NOUN	_	Gender=Masc|Number=Plur	44	conj	O	_
 48	e	e	CCONJ	_	_	49	cc	O	_
 49	aves	ave	NOUN	_	Gender=Fem|Number=Plur	44	conj	O	_
 50	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	52	nsubj	O	_
@@ -281331,7 +281331,7 @@
 14	,	,	PUNCT	_	_	17	punct	O	_
 15	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	17	case	O	_
 16	como	como	ADP	_	_	15	fixed	O	_
-17	peixes	peixe	NOUN	_	Gender=Masc|Number=Plur	13	nmod:appos	O	_
+17	peixes	peixe	NOUN	_	Gender=Masc|Number=Plur	5	nmod:appos	O	_
 18	,	,	PUNCT	_	_	19	punct	O	_
 19	tartarugas	tartaruga	NOUN	_	Gender=Fem|Number=Plur	17	conj	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
@@ -284365,7 +284365,7 @@
 63	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	64	det	O	_
 64	problemas	problema	NOUN	_	Gender=Masc|Number=Plur	62	nsubj	O	_
 65	quanto	quanto	ADP	_	_	66	case	O	_
-66	ignição	ignição	NOUN	_	Gender=Fem|Number=Sing	64	nmod:appos	O	_
+66	ignição	ignição	NOUN	_	Gender=Fem|Number=Sing	64	nmod	O	_
 67-68	do	_	_	_	_	_	_	_	_
 67	de	de	ADP	_	_	69	case	O	_
 68	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	69	det	O	_
@@ -288152,7 +288152,7 @@
 15	como	como	ADV	_	_	14	fixed	O	_
 16	:	:	PUNCT	_	_	18	punct	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
-18	centrifugação	centrifugação	NOUN	_	Gender=Fem|Number=Sing	9	appos	O	_
+18	centrifugação	centrifugação	NOUN	_	Gender=Fem|Number=Sing	11	nmod:appos	O	_
 19	ou	ou	CCONJ	_	_	20	cc	O	_
 20	flotação	flotação	NOUN	_	Gender=Fem|Number=Sing	18	conj	O	_
 21	,	,	PUNCT	_	_	22	punct	O	_
@@ -289485,7 +289485,7 @@
 # sent_id = 243-20141015-MONOGRAFIA_1-87
 1	Contudo	contudo	ADV	_	_	21	advmod	O	_
 2	,	,	PUNCT	_	_	1	punct	O	_
-3	como	como	ADV	_	_	11	advmod	O	_
+3	como	como	SCONJ	_	_	11	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
 5	densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
 6-7	do	_	_	_	_	_	_	_	_
@@ -290410,7 +290410,7 @@
 38	tais	tal	DET	_	Gender=Masc|Number=Plur|PronType=Dem	41	case	O	_
 39	como	como	ADP	_	_	38	fixed	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
-41	Na2+	Na2+	PROPN	_	Gender=Masc|Number=Sing	24	nmod:appos	O	_
+41	Na2+	Na2+	PROPN	_	Gender=Masc|Number=Sing	29	nmod:appos	O	_
 42	.	.	PUNCT	_	_	4	punct	O	_
 
 # text = As resinas possuem limites para a troca iônica, ficando saturadas no referido metal.
@@ -292318,7 +292318,7 @@
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
 15	solubilização	solubilização	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
 16	como	como	ADP	_	_	17	case	O	_
-17	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	15	nmod:appos	O	_
+17	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	10	nmod	O	_
 18	capazes	capaz	ADJ	_	Gender=Fem|Number=Plur	17	amod	O	_
 19	de	de	ADP	_	_	20	mark	O	_
 20	predizer	predizer	VERB	_	VerbForm=Inf	17	acl	O	_
